src/App.tsx

```diff
@@ -1,61 +1,136 @@
 import './App.scss';
+import { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+
+function getUser(userId: number) {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser;
+}
+
+const initialTodos: Todo[]
+  = todosFromServer.map(
+    todo => ({
+      ...todo, user: getUser(todo.userId),
+    }),
+  );
```

Comment:
```suggestion
const initialTodos: Todo[] = todosFromServer.map(todo => ({
  ...todo,
  user: getUser(todo.userId),
}));
```
Will look a bit better